sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 # Release History
 
+## 8.1.1 (Unreleased)
+
+- Exposes a BatchClient factory method
+
 ## 8.1.0 (Unreleased)
+
 ### Features
+
 - Adds support for task slots
-    - `JobOperations.getTaskSlotCounts()` returns task slot counts
-    - `JobOperations.getTaskResult()` returns a `TaskCountsResult` object containing both task and slot counts
+  - `JobOperations.getTaskSlotCounts()` returns task slot counts
+  - `JobOperations.getTaskResult()` returns a `TaskCountsResult` object containing both task and slot counts
 
 ## 8.1.0-beta.1 (Unreleased)
```

Comment:
nit: This line looks like it can be removed

Reply:
Will fix.